docs/static/Devel/Developers.md

```diff
@@ -38,30 +38,31 @@ consists in
-| `npm run test`           | Build the 3 packages and tests shared, browser scripts and examples html. Uses [this test script](./bin/test.js)                                                                   |
-| `npm run assert-code`    | Run `npm run eslint` and `npm run test`. Also ran by CI. See [here](#continuous-integration-travis-ci) for more information                                                        |
-| `npm run validate-links` | Run this [script](bin/../../../../bin/remarkValidateLinks.js) to detect dead links in markdown. Also ran by CI. See [here](#continuous-integration-travis-ci) for more information |
-| `npm run pre-publish`    | Change version in all package.json ( eg `npm run pre-publish x.x.x` ). See [this script](./bin/prePublish.js)                                                                      |
-| `npm run docs-shared`    | Generate the [JSDOC shared package documentation](./docs/jsdocConfig/jsdoc.shared.json)                                                                                            |
-| `npm run docs-browser`   | Generate the [JSDOC browser package documentation](./docs/jsdocConfig/jsdoc.browser.json)                                                                                          |
-| `npm run docs-node`      | Generate the [JSDOC node package documentation](./docs/jsdocConfig/jsdoc.node.json)                                                                                                |
-| `npm run docs-home`      | Generate the [JSDOC documentation landing page](./docs/jsdocConfig/jsdoc.home.json)                                                                                                |
-| `npm run docs`           | Run `npm run docs-shared`, `npm run docs-browser`, `npm run docs-node`, and `npm run docs-home`                                                                                    |
-| `npm run dev-docs`       | Launch a watcher for generating and debugging the documentation. See [here](./docs/Readme.md) for more information                                                                 |
-| `npm run host`           | host the bundle with an [ExpressAppWrapper](./bin/host.js). <br>http://locahost:8000/                                                                                              |
-| `npm run start`          | Run `npm run build-browser` and `npm run host`                                                                                                                                     |
+| `npm run reset`          | Reinstalls npm dependencies. This script runs `npm run clean` and `npm install` command                                                                                                                                                                               |
```

Comment:
Why is there so much noise with many lines that appear as removed and then replaced with apparently identical content (at least on github diff view) ?

Reply:
git diff introduce this and went wrong